models/ed/R/model2netcdf.ED2.R

```diff
@@ -21,25 +22,155 @@
 ##'
 ##' @author Michael Dietze, Shawn Serbin, Rob Kooper, Toni Viskari, Istem Fer
 ## modified M. Dietze 07/08/12 modified S. Serbin 05/06/13
-model2netcdf.ED2 <- function(outdir, sitelat, sitelon, start_date, end_date) {
-
-  flist <- dir(outdir, "-T-")
-  if (length(flist) == 0) {
-    print(paste("*** WARNING: No tower output for :", outdir))
+model2netcdf.ED2 <- function(outdir, sitelat, sitelon, start_date, end_date, pft_names = NULL, dbh_breaks = 0) {
+
+  start_year <- lubridate::year(start_date)
+  end_year   <- lubridate::year(end_date) 
+  
+  flist <- list()
+  flist[["-T-"]] <- dir(outdir, "-T-") # tower files
+  flist[["-E-"]] <- dir(outdir, "-E-") # monthly files
+  
+  # check if there are files
+  file.check <- sapply(flist, function (f) length(f) != 0)
+  
+  if (!any(file.check)) {
+    
+    # no output files
+    print(paste("*** WARNING: No output files found for :", outdir))
```

Comment:
warnings should use use the PEcAn logger warning function, not print

Reply:
these are remnants from the current code, will change